File: fixtures/scale/csharp/large/Families/Atbash/AtbashVectors.cs
using System.Collections.Generic;
using CipherLab.Abstractions;

namespace CipherLab.Families.Atbash
{
    public static class AtbashVectors
    {
        public static List<TestVector> All()
        {
            return new List<TestVector>
            {
                new TestVector("ATTACKATDAWN", "FBEOTEXTGGFZ"),
                new TestVector("THEQUICKBROWNFOX", "YPPELCZKEXXICXJV"),
                new TestVector("DEFENDTHEEASTWALL", "IMQSEXQHHKJEIOVJM"),
            };
        }
    }
}

File: fixtures/scale/php/large/src/Families/Atbash/AtbashVectors.php
<?php

namespace CipherLab\Families\Atbash;

use CipherLab\Abstractions\TestVector;

// Known-answer test data captured from AtbashCipher's own encode().
class AtbashVectors
{
    /** @return TestVector[] */
    public static function all(): array
    {
        return [
            new TestVector("ATTACKATDAWN", "FBEOTEXTGGFZ"),
            new TestVector("THEQUICKBROWNFOX", "YPPELCZKEXXICXJV"),
            new TestVector("DEFENDTHEEASTWALL", "IMQSEXQHHKJEIOVJM"),
        ];
    }
}
